chainer/functions/normalization/batch_normalization.py

```diff
@@ -27,100 +60,266 @@ def check_type_forward(self, in_types):
             gamma_type.shape == beta_type.shape,
         )
 
-        if len(in_types) == 5:
-            mean_type, var_type = in_types[3:]
-            type_check.expect(
-                mean_type.dtype == x_type.dtype,
-                mean_type.shape == gamma_type.shape,
-                var_type.dtype == x_type.dtype,
-                var_type.shape == gamma_type.shape,
-            )
-
     def forward(self, inputs):
         xp = cuda.get_array_module(*inputs)
-        x, gamma, beta = inputs[:3]
+        x, gamma, beta = inputs
```

Comment:
This function was never documented, though. Is it okay to remove the optional arguments as long as I do not brake any of the documented APIs?